config/routes.rb

```diff
@@ -1,3 +1,13 @@
-Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/'
+  mount Rswag::Api::Engine => '/api-docs'
+  devise_for :users, defaults: { format: :json }, path: 'users',
+                     path_names: { sign_in: 'login', sign_out: 'logout', registration: 'signup' }
+
+  namespace :api, defaults: { format: :json } do
+    namespace :v1 do
+      resources :rooms, only: %i[index create show destroy]
+      resources :reservations, only: %i[index show create destroy]

```

Comment:
- [ ] Like I pointed above, you aren't using the `show` action here, so I would remove the route.